### DISCUSSION TITLE
NFT MARKETPLACE IN ZKSYNC ERA

### DISCUSSION THREAD

**Discussion by jayson2381** [zkSync-Community-Hub/zksync-developers#349] [Feature requests]
### Network

Both

### Provide a brief description of the feature you would like to see implemented.

ZKSYNC IS THE ENDGAME

### Who does this feature request benefit the most (users, developers, protocol, etc.)?

LETS GO TO MOON

**Comment by jayson2381** [comment]
LAMBO SOON

**Comment by 0xsisyfos** [comment]
There are a few NFT marketplace available. You can check the app on zkSync Era on DappRadar: https://zksync.dappradar.com/
